gc/base/EnvironmentBase.hpp

```diff
@@ -689,6 +689,15 @@ class MM_EnvironmentBase : public MM_BaseVirtual
 	MMINLINE MM_HeapRegionQueue *getRegionLocalFull() const { return _regionLocalFull; }
 #endif /* OMR_GC_SEGREGATED_HEAP */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * Reinitialize the env's language specific object buffers based on the the number of restore GC threads.
+	 *
+	 * @return boolean indicating whether thread buffers were reinitialized successfully.
```

Comment:
`@param/@return` lines shouldn't end with a period
```suggestion
	 * @return boolean indicating whether thread buffers were reinitialized successfully
```